src/main/java/de/uni_tuebingen/ub/nppm/servlet/backend/AdministrationAuswahlServlet.java

```diff
@@ -4,16 +4,23 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-public class AdministrationAuswahlServlet extends AbstractBackendServlet { 
-    
+public class AdministrationAuswahlServlet extends AbstractBackendServlet {
+
     @Override
     protected String getTitle() {
         return "administration";
     }
 
     @Override
     protected void generatePage(HttpServletRequest request, HttpServletResponse response) throws Exception {
-        RequestDispatcher rd = request.getRequestDispatcher("admin.auswahlfelder.jsp");
-        rd.include(request, response);
+          //Use isAdminRequired to check if the user is an administrator.
```

Comment:
This check will be done by the parent function. The generatePage function should stay as before, you should only override the isAdminRequired function.

Reply:
done: look at commit: Administration